components/widgets/PropertyWidget.php

```diff
@@ -69,11 +83,6 @@ public function init() {
         }
         $this->propertyFormatters = $formatters;
 
-        // must be not null
-        if ($this->uri === null) {
```

Comment:
why did you removed this check ? Is it gonna be ok with the older functionalities using it ?

Reply:
I re-write this check (in specific cases because it is only relevant in the infrastructure view)